src/main/java/de/codecrafter47/taboverlay/config/placeholder/CustomPlaceholderColorAnimation.java

```diff
@@ -87,6 +89,7 @@ private void updateReplacement() {
             TextColor b = colors.get((ia + 1) % colors.size());
             TextColor c = TextColor.interpolateSine(a, b, sd - ia);
             sb.append(c.getFormatCode());
+            sb.append(formats);
```

Comment:
You should change line 83 to ` * (9 + formats.length())`

Reply:
You mean `new StringBuilder(text.length() * (9 + formats.length())`?

Reply:
Updated it. Hope I understood it correctly.